### DISCUSSION TITLE
How to create a Video Sitemap?

### DISCUSSION THREAD

**Discussion by ghost** [mediacms-io/mediacms#549] [General]
I was able to generate a sitemap for the static pages using [django.contrib.sitemap](url) but has anyone been able generate a dynamic video sitemap for all of their media pages? I'm not much of a developer so anything to point me in the right direction would be helpful. The django tutorials I found are for blog and article sitemaps.

**Comment by mgogoulos** [comment]
hi, that's possible, would require a query to the database to get all media pages and then prepare an html with them - and of course keep in mind that this will be an expensive query  on the database if the number of media pages grows big

**Comment by ghost** [comment]
thanks! i decided to try uploading the /rss page as a sitemap to the google search console and it worked perfectly. i simply increased the `return media` amount in feeds.py for larger coverage.
